src/hud.cpp

```diff
@@ -87,24 +87,31 @@ Hud::Hud(video::IVideoDriver *driver, scene::ISceneManager* smgr,
 	m_halo_boxes.clear();
 
 	m_selection_pos = v3f(0.0, 0.0, 0.0);
-	std::string mode = g_settings->get("node_highlighting");
+	std::string mode_setting = g_settings->get("node_highlighting");
```

Comment:
Suuuuuuper Needed for ascetics and video production alike... Loads more Minetest related content on YouTube may surface due to the option being present. Some art is designed in great detail and the Black lines or alt. color upon closer inspection make that ART look terrible. I think everyone will agree once you get past the rainbow wool hut stage  it's nothing but art from there and as time passes it gets more and more intricate. People want to show off their art and some will go so far as to prominently display it in the largest way possible via pubic accessible posting or videos. But only if the "close inspection" is up to par with the entirety of the workings itself. This may seem like an Odd observation but I am an odd person in all reality and feel that this is a great addition to the Minetest world. Another Great suggestion by the Great Ezhh :